oak-store-document/src/main/java/org/apache/jackrabbit/oak/plugins/document/DocumentNodeStoreService.java

```diff
@@ -225,6 +236,8 @@ static DocumentStoreType fromString(String type) {
     private Feature docStoreThrottlingFeature;
     private Feature noChildOrderCleanupFeature;
     private Feature cancelInvalidationFeature;
+    private Feature docStoreFullGCFeature;
+    private Feature docStoreEmbeddedVerificationFeature;
```

Comment:
Is this Feature ever initialized?

Reply:
It seems to be a miss, will initialize this feature.

Reply:
Addressed in https://github.com/apache/jackrabbit-oak/pull/1482/commits/6b699d245d42d4272e5d7ba4e833d195e7586706